FreeRTOS/Demo/CORTEX_MPU_M7_NUCLEO_H743ZI2_GCC_IAR_Keil/ST_Code/Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_sd_ex.h

```diff
@@ -1,113 +0,0 @@
-/**
```

Comment:
Is this meant to be checked in?

Reply:
No, I do not think so. I did not intend to add it.